docs/_docs/oauth.md

```diff
@@ -88,10 +88,17 @@ If you ever need to reference Zapier's redirect URL inside your Zapier integrati
 
 {% raw %}`{{bundle.inputData.redirect_uri}}`{% endraw %}
 
+<a id="PKCE"></a>
+## Add PKCE Support
+
+![Zapier OAuth v2 with PKCE Extension])(https://cdn.zappy.app/124a3c00d8bc37dadd953f19451205a5.png)
+
+Zapier provides built-in support for [PKCE](https://oauth.net/2/pkce/#credentials) (Proof Key for Code Exchange and pronounced "pick-see"), an extension to the authorization code flow that adds a layer of protection against security vulnerabilities. The code generation and exchange steps of the flow occur automatically by Zapier when enabled.
+
 <a id="credentials"></a>
 ## Add Application Credentials to Zapier
 
-![Add application credentials to Zapier](https://cdn.zappy.app/143501610dce552f7ce003dc6632dc2b.png)
+![Add application credentials to Zapier](https://cdn.zappy.app/1664c3f9e9005d83b9fcfc3f6c5e7ecc.png)
```

Comment:
@michellechu77 This image seems to be broken for me:
![](https://cdn.zappy.app/ab1d632149fb5dd869e052064f3b07fd.png)

Reply:
Updated! TY, great catch.